addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -47,7 +47,7 @@ wTourUtils.registerWebsitePreviewTour('edit_link_popover', {
     {
         content: "Type the link URL /contactus",
         trigger: '#o_link_dialog_url_input',
-        run: 'text /contactus'
+        run: "edit /contactus && blur we-title.o_short_title",
```

Comment:
Is the blur really needed here and at line 66 + 106?
Won't it be automatically triggered when clicking on the footer or the save button in the next step?

Reply:
You're right ! I'll remove all modifications in popover tour. There are leftovers of my tests to debug this tour...